pkgs/development/python-modules/ionoscloud/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, urllib3
+, six
+, certifi
+, python-dateutil
+, asn1crypto
+
+, pytest
+, pytest-cov
+, pytest-randomly
+}:
+
+buildPythonPackage rec {
+  pname = "ionoscloud";
+  version = "6.1.8";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash= "sha256-QQNDqKZ+X+w5vHFXr793QqRjShNBZ182DyD3fUUxqkc=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [ urllib3 six certifi python-dateutil asn1crypto ];
+
+  buildInputs = [ pytest pytest-cov pytest-randomly ];
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook pytest-randomly ];
```
Could you please use `pytestCheckHook` instead?
Also, we do not need to measure coverage because we test to ensure the package works.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook

---

Turns out, upstream only has coverage testing, but no actual unit or integration tests to go with that. Coverage just returns 0% 🥴